### PR TITLE
Add sunkit-image SunPy idea

### PR DESCRIPTION
**Pull request by Cadair**
ping @nabobalis @wafels

### CODE REVIEW

File: _projects/2017/sunpy/sunkit-image.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+1. Port the Multi-Scale Gaussian Normalisation code from [#1899](https://github.com/sunpy/sunpy/pull/1899).
+2. Convert the [differential rotation code](https://github.com/sunpy/sunpy/blob/master/sunpy/physics/differential_rotation.py) in SunPy to use `sunpy.coordinates`.
+3. Implement image warping for solar differential rotation. [#1876](https://github.com/sunpy/sunpy/pull/1876).
+4. Implement the [OCCULT-2 algorithm](http://arxiv.org/abs/1307.5046) for coronal loop tracing.
+5. Implement running and base difference functionality and the persistence transform. See Figure 2 in [this paper](http://iopscience.iop.org/article/10.1088/0004-637X/736/2/102/pdf) for some ideas.
+
+optional extras:
+
+6. Refactor and write a Python wrapper for [FLCT](https://arxiv.org/abs/0712.4289) [code](http://solarmuri.ssl.berkeley.edu/overview/publicdownloads/software.html).
+8. Implement image alignment using feature detection and tracking. [Example](http://scikit-image.org/docs/dev/auto_examples/features_detection/plot_brief.html)
```
Comment (nabobalis) [inline _projects/2017/sunpy/sunkit-image.md]: Not sure how: http://image-registration.readthedocs.io/en/latest/ will fit in. If we could add that or do something else with it.

**Comment by wafels** [comment]
Existing map co-registration in sunpy relies on scikit image directly,
which carries such functionality.  This was done to avoid the
"roll-your-own" approach.

**Comment by nabobalis** [comment]
But does it do real alignment?

**Comment by Cadair** [comment]
I has forgotten that existed.

**Comment by wafels** [comment]
What do you mean by real alignment?

**Pull request merged**